graphql/src/main/java/com/scalar/db/graphql/GraphQlFactory.java

```diff
@@ -0,0 +1,139 @@
+package com.scalar.db.graphql;
+
+import com.google.common.collect.ImmutableList;
+import com.scalar.db.api.DistributedStorage;
+import com.scalar.db.api.DistributedStorageAdmin;
+import com.scalar.db.api.DistributedTransactionManager;
+import com.scalar.db.exception.storage.ExecutionException;
+import com.scalar.db.graphql.schema.CommonSchema;
+import com.scalar.db.graphql.schema.TableGraphQlModel;
+import com.scalar.db.service.StorageFactory;
+import com.scalar.db.service.TransactionFactory;
+import graphql.GraphQL;
+import graphql.Scalars;
+import graphql.schema.GraphQLCodeRegistry;
+import graphql.schema.GraphQLFieldDefinition;
+import graphql.schema.GraphQLObjectType;
+import graphql.schema.GraphQLSchema;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class GraphQlFactory {
```

Comment:
This class looks immutable so can you add @Immutable to the class? If not, please add an appropriate annotation like @ThreadSafe or @ThreadNotSafe.
We don't use any tools that utilize the annotations but it's helpful for users and developers to grasp what a class is designed for.

Also, for other classes, it would be nice if we have the annotations.

Reply:
Thank you! Added `@Immutable` to this class and the `TableGraphQlModel` class in 6271071.
For other classes, I'm not fully sure but I don’t think it is needed because they are helper classes.